pkg/azure/azure.go

```diff
@@ -0,0 +1,41 @@
+package azure
```

Comment:
would need tests for this package

Reply:
Yes, i would add this in the next commits. Wanted to get the business logic reviewed.